### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by igloowhite**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @igloowhite-student

### CODE REVIEW

File: Dictionary.java [hunk 1]
```diff
@@ -1,64 +1 @@
...
-        for(String word : words){
-            if(word.length() == numLetters){
-                matchWords.add(word);
-            }
-        }
-
-        int rand = (int)(Math.random() * matchWords.size());
-        return matchWords.get(rand);
-       
-    }
```
Comment (igloowhite) [inline Dictionary.java:62]: This is not a good way to do this

Comment (igloowhite-student) [reply Dictionary.java:62]: I'm still not sure what you mean

File: Dictionary.java [hunk 2]
```diff
@@ -1,64 +1 @@
...
- * @version 11/25/2020
- */
-import java.util.ArrayList;
-
-public class Dictionary{
-    
-    private static String[] words = {"abstract","primitive","object",
-            "array","annoy","ignore","prefer","attention","instead","problem",
-            "calm","investigate","protect","comfortable","invite","proud",
-            "consequences","important","question","curious","jealous",
```
Comment (igloowhite) [inline Dictionary.java:15]: `
This
that
other
`

File: Dictionary.java [hunk 3]
```diff
@@ -1,64 +1 @@
...
-            "reminds","curve","leader","repeat","decide","list","report",
-            "directions","listen","rhyme","discover","lovely","respect",
-            "disappointed","measuring","searching","embarrassed","miserable",
-            "special","enormous","mumble","spotless","exhausted","negative",
-            "squirm","explore","nervous","stomped","fair","nibbled","suddenly",
-            "fascinating","note","suggestion","feast","notice","surprise",
-            "focus","observing","uncomfortable","frustrated","opposite",
```
Comment (igloowhite) [inline Dictionary.java:22]: No no no.

File: Dictionary.java [hunk 4]
```diff
@@ -1,64 +1 @@
...
-            }
-        }
-
-        int rand = (int)(Math.random() * matchWords.size());
-        return matchWords.get(rand);
-       
-    }
-
-}
+I did some work!
```
Comment (igloowhite) [inline Dictionary.java]: I don't like this
```
try this instead
```

Comment (igloowhite) [inline Dictionary.java]:
```suggestion
This would be better
```

**Pull request merged**